examples/create_rosa_sts_cluster/oidc_configuration/cluster_with_managed_oidc_config/README.md

```diff
@@ -22,39 +22,42 @@ This Terraform example creates a ROSA STS cluster that uses a managed OIDC confi
         ```
         export TF_VAR_account_role_prefix=<account_role_prefix>
         ```
-    2. The availability zones should be a zone within your AWS region, which you chose with the `TF_VAR_cloud_region` variable. This value must be enclosed by double quotes, a bracket pair, and single quotes.     
+    1. The availability zones should be a zone within your AWS region, which you chose with the `TF_VAR_cloud_region` variable. This value must be enclosed by double quotes, a bracket pair, and single quotes.
```

Comment:
Why is it 1 1 1 , instead of the list 1, 2, 3...?

Reply:
Markdown auto-generates the numbers.